fe/fe-core/src/main/java/com/starrocks/catalog/FunctionSet.java

```diff
@@ -562,6 +562,8 @@ private void initAggregateBuiltins() {
         // Sum
         String[] sumNames = {"sum", "sum_distinct"};
         for (String name : sumNames) {
+            addBuiltin(AggregateFunction.createBuiltin(name,
```

Comment:
Why do you move double to the first one?
Is this will make some query select sum(double) other than sum(bigint)?

Reply:
We need to sum(varchar) match sum(double)

No. sum(bigint) will still match sum(bigint), because which is identical

Reply:
which function will be matched when parsing `sum(int)`?

Reply:
sum(int)